### ISSUE TITLE
Add booking system features to gain users

### ISSUE THREAD

**Issue by Rupayan20** [NKumar141/railgadi#9]
If we can add train booking facility along with train location sharing to this platform then users will be attractive to it apart from other applications.

And using UTS and IRCTC API we can book train's tickets through our application

**Comment by NKumar141** [comment]
yes , booking is one of the features we can pursue in the later stages of the application . Actually we have to approach this project in phases . From one phase to another we can keep adding features to gain users . but it is important to first finish off all the features of the previous phase !
